### ISSUE TITLE
Not able to change in View of Tinker

### ISSUE THREAD

**Issue by suhasadhav** [botman/botman#675]
- BotMan Version: #.#.#
- PHP Version:
- Messaging Service(s):
- Cache Driver:

### Description:
I want to add HTML replies from botman but not able to add due to vue.js escape even I added below line in BotManTinker.vue file but changes are not reflected. 

              <p class="ChatLog__message" v-html="message.text"></p>

How can I achieve HTML replies in Tinker?

**Comment by mpociot** [comment]
Hello,

this is not directly related to BotMan.
You need to run `npm install` and `npm run dev` in order to rebuild your `app.js` file

**Comment by myrsk** [comment]
Tried to npm install and npm run dev, but doesnt seem to sort it out ...

**Comment by choc-design** [comment]
I started yesterday with a `botman new project` and neither npm install, run dev, watch, etc. fixed the problem. Until i found the 'solution'.

The botman-tinker - TinkerComponent that was used everytime i rebuilded the app.js, is located in the `node_modules/botman-tinker/src/components` folder. 

I copied this file to my `resources/assets/js/components` folder and included it the 'normal' way. After that everything worked like expected (;

![bildschirmfoto 2018-04-03 um 19 33 44](https://user-images.githubusercontent.com/1814679/38265556-89e80db6-3776-11e8-958f-9bdabcbaaab1.png)
